### ISSUE TITLE
Image Pull Error

### ISSUE THREAD

**Issue by jabbasj** [open-webui/helm-charts#35]
Normal   Pulling                 33s   kubelet                                Pulling image "ghcr.io/ollama/open-webui:main"
  Warning  Failed                  3s    kubelet                                Failed to pull image "ghcr.io/ollama/open-webui:main": rpc error: code = DeadlineExceeded desc = failed to pull and unpack image "ghcr.io/ollama/open-webui:main": failed to resolve reference "ghcr.io/ollama/open-webui:main": failed to do request: Head "https://ghcr.io/v2/ollama/open-webui/manifests/main": dial tcp 142.182.19.19:443: i/o timeout
  Warning  Failed                  3s    kubelet                                Error: ErrImagePull
  Normal   BackOff                 2s    kubelet                                Back-off pulling image "ghcr.io/ollama/open-webui:main"
  Warning  Failed                  2s    kubelet                                Error: ImagePullBackOff

**Comment by 0xThresh** [comment]
Hi @jabbasj, can you please send your `values.yaml` file?

**Comment by 0xThresh** [comment]
I just noticed you're trying to pull `ghcr.io/ollama/open-webui:main`. You're trying to pull an Open WebUI image from Ollama's repo, which is not where it lives. I've checked the full Helm chart repo and confirmed that the string `ghcr.io/ollama/open-webui:main` does not appear in it, so it seems like this is a bad value being added in your values that you're passing to the build process.

The default entry in `values.yaml` should cover your needs:
```
image:
  repository: ghcr.io/open-webui/open-webui
  tag: "latest"
  pullPolicy: Always
```
